### PR TITLE
Fix #188, //rel.union accepts set of sets

### LINKED ISSUE #188: //rel.union needs to take generic set
### Current arrai Version
Currently `//rel.union` only takes ordered set, it needs to take in generic set.

### Use-cases
```
@> //rel.union({{1, 2}, {2, 3}, {3}})
{1, 2, 3}
```

### PR DESCRIPTION
**Pull request by nofun97**
Fixes #188  .

Changes proposed in this pull request:
- `//rel.union` can now take an array of sets or set of sets

Checklist:
- [x] Added related tests
- [x] Made corresponding changes to the documentation

### CODE REVIEW

File: syntax/std_rel.go [hunk 1]
```diff
@@ -9,7 +9,18 @@ func stdRel() rel.Attr {
 		rel.NewNativeFunctionAttr("union", func(v rel.Value) rel.Value {
 			s := v.(rel.Set)
 			sets := make([]rel.Set, 0, s.Count())
-			for e, ok := s.ArrayEnumerator(); ok && e.MoveNext(); {
+			var e rel.ValueEnumerator
+			switch u := v.(type) {
+			case rel.Array:
```
Comment (marcelocantos) [inline syntax/std_rel.go:14]: Only sets should be accepted.

Comment (nofun97) [reply syntax/std_rel.go:14]: treating array as set created complications as `Enumerator.Current()` would return a tuple `(@, @values)`

Comment (marcelocantos) [reply syntax/std_rel.go:14]: That's as intended.

Comment (nofun97) [reply syntax/std_rel.go:14]: right, some of the example uses `//rel.union(array)`, I'll change it

**Pull request merged**